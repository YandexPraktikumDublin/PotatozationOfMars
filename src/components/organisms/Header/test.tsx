import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Header } from '.'

describe('<Header />', () => {
  it('should renders correct <Header />', () => {
    const wrapper = shallow(<Header />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
