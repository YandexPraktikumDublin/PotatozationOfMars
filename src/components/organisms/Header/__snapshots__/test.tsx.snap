// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should renders correct <Header /> 1`] = `
<header
  className="p-2 items-center w-full flex"
>
  <div
    className="my-auto"
  >
    <Pause />
  </div>
  <div
    className="flex ml-auto mr-0"
  >
    <Navigation />
    <TumblerTheme />
  </div>
</header>
`;
